### ISSUE TITLE
W32/mingw builds do not catch exceptions

### ISSUE THREAD

**Issue by umlaeute** [umlaeute/Gem#225]
instead Pd is terminated...

**Comment by Spacechild1** [comment]
Does this only happen with 32-bit builds? If yes, it may be this: https://git.iem.at/pd/vstplugin/-/tree/master#warning-about-32-bit-mingw

**Comment by umlaeute** [comment]
probably, however, I thought I', already are building with `-static-libstdc++` in https://git.iem.at/pd/Gem/-/jobs/41776#L885

it appears, though, that i'm building with *both* `-lstdc++` and `-static-libstdc++` which might explain why it's not working.
not that some of the dependencies as installed by pacman (i think FTGL is to blame) also dynamically link against `-lstdc++`, which complicates the issue.

**Comment by Spacechild1** [comment]
> it appears, though, that i'm building with both -lstdc++ and -static-libstdc++ which might explain why it's not working.

Yes, it looks like `Gem.dll` is statically linked, but all the plugins are dynamically linked.

> not that some of the dependencies as installed by pacman (i think FTGL is to blame) also dynamically link against -lstdc++, which complicates the issue.

Yep, that could be problem. In the worst case, you'd have to compile `ftgl` yourself.

**Comment by umlaeute** [comment]
> Yes, it looks like Gem.dll is statically linked, but all the plugins are dynamically linked.

doh.
i'm going to check.
